projectile.el

```diff
@@ -358,6 +358,12 @@ containing a root file."
   :group 'projectile
   :type '(repeat function))
 
+(defcustom projectile-dirconfig-comment-prefix
+  nil
```

Comment:
I guess we should set this to something by default.

Reply:
Hmm I thought it should default to nil (that is, disabled) unless a user sets it themselves. To avoid surprising someone with existing ignore directives. Eg if we made this var ‘#’ by default, someone with an existing line in their projectile file like:

    #*

... would suddenly lose that ignore functionality.

Reply:
Yeah, that makes sense.